### PR TITLE
Added option for thumbnail only

### PR DESCRIPTION
**Pull request by 1RandomDev**
I've added an option which downloads the thumbnail of a video only and saves it as JPG.

### CODE REVIEW

**Comment by kimpig** [comment]
When I clicked on the file name, it links to "Filename.webm", not "Filename.jpg".
So 404 error occurs.
Is there a solution?

제목 없음

**Comment by PikuZheng** [comment]
> When I clicked on the file name, it links to "Filename.webm", not "Filename.jpg". So 404 error occurs. Is there a solution?
> 
> 제목 없음

try this:
https://github.com/PikuZheng/metube/commit/b56d3f0c836b043cb2b39dec9208db1c8306398d

**Comment by kimpig** [comment]
> > When I clicked on the file name, it links to "Filename.webm", not "Filename.jpg". So 404 error occurs. Is there a solution?
> > 제목 없음
> 
> try this: [PikuZheng@b56d3f0](https://github.com/PikuZheng/metube/commit/b56d3f0c836b043cb2b39dec9208db1c8306398d)

This links to "FileName.webm.jpg"

**Comment by PikuZheng** [comment]
> This links to "FileName.webm.jpg"

.webm.jpg is planned, but the real name is .webm.jpg.jpg...let me try again.

**Comment by 1RandomDev** [comment]
Not the most elegant solution but should be fixed in 0f27bf854560fa37a5fa9f9d9645fe4c321fe487
